<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/rucktrack.qrc">
    <normaloff>:/resources/icon.png</normaloff>:/resources/icon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="configPageSelector">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>101</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">::item { width:87; }
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="iconSize">
      <size>
       <width>64</width>
       <height>64</height>
      </size>
     </property>
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/rucktrack.qrc">
        <normaloff>:/resources/setup-general.png</normaloff>:/resources/setup-general.png</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Maps</string>
      </property>
      <property name="icon">
       <iconset resource="../resources/rucktrack.qrc">
        <normaloff>:/resources/setup-maps.png</normaloff>:/resources/setup-maps.png</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsUserCheckable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QStackedWidget" name="configPages">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalSettingsPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="generalTitleLabel">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>General settings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>User interface</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <widget class="QCheckBox" name="showProgressBar">
            <property name="text">
             <string>Show progress bar</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="dropFaultyTrackPoints">
            <property name="text">
             <string>Drop track points with wrong timestamps</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Paths</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Cache directory:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="cacheDirectory"/>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="cacheDirectoryBtn">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources/rucktrack.qrc">
              <normaloff>:/resources/document-open.png</normaloff>:/resources/document-open.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QCheckBox" name="useSystemCacheDirectory">
            <property name="text">
             <string>Use system cache directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mapsSettingsPage">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>16</number>
       </property>
       <item>
        <widget class="QLabel" name="generalTitleLabel_4">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Maps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Map provider</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QComboBox" name="mapProviderComboBox"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Caching</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="4">
           <widget class="QCheckBox" name="enableCaching">
            <property name="text">
             <string>Enable on-disk caching of map tiles</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Maximum cache size:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="cacheSizeSpinBox">
            <property name="minimum">
             <number>10</number>
            </property>
            <property name="maximum">
             <number>500</number>
            </property>
            <property name="singleStep">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>MB</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>48</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/rucktrack.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configPageSelector</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>configPages</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useSystemCacheDirectory</sender>
   <signal>toggled(bool)</signal>
   <receiver>cacheDirectory</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useSystemCacheDirectory</sender>
   <signal>toggled(bool)</signal>
   <receiver>cacheDirectoryBtn</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>225</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
